### PR TITLE
add authors, unify nereids, fix word order

### PR DESCRIPTION
**Pull request by celerini**
@MultiSeptim, please check this.

Eric, please wait until @MultiSeptim approve or reject my changes.

### CODE REVIEW

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 1]
```diff
@@ -1,17 +1,19 @@
+# Septim, 2022
```
Comment (MultiSeptim) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
# Petr Šlejhar (Septim), 2022
```

**Review by MultiSeptim — CHANGES_REQUESTED** [review]

**Comment by MultiSeptim** [comment]
Jinak OK.

**Comment by celerini** [comment]
Eric, you can merge it. Thanks.

**Comment by cooljeanius** [comment]
> Eric, you can merge it. Thanks.

OK, merged!

**Pull request merged**